metadata-ingestion/docs/sources/iceberg/iceberg.md

```diff
@@ -0,0 +1,23 @@
+### Concept Mapping
+
+<!-- This should be a manual mapping of concepts from the source to the DataHub Metadata Model -->
+<!-- Authors should provide as much context as possible about how this mapping was generated, including assumptions made, known shortcuts, & any other caveats -->
+
+This ingestion source maps the following Source System Concepts to DataHub Concepts:
+
+<!-- Remove all unnecessary/irrevant DataHub Concepts -->
+
+| Source Concept | DataHub Concept | Notes |
+| -- | -- | -- |
+| `iceberg` | [Data Platform](docs/generated/metamodel/entities/dataPlatform.md) | |
+| Table | [Dataset](docs/generated/metamodel/entities/dataset.md) | Each Iceberg table maps to a Dataset named using the parent folders.  If a table is stored under `my/namespace/table`, the dataset name will be `my.namespace.table`.  If a [Platform Instance](https://datahubproject.io/docs/platform-instances/) is configured, it will be used as a prefix: `<platform_instance>.my.namespace.table`. |
+| [Table property](https://iceberg.apache.org/docs/latest/configuration/#table-properties) | [User (a.k.a CorpUser)](docs/generated/metamodel/entities/corpuser.md) | A configurable table property can be used as a source of CorpUser owner. |
```

Comment:
Question: Is the property the normal `[U|u]ser` or `CorpUser` when applied to an Iceberg table?

Reply:
@kbendick  I'm not sure I understand your question here Kyle...  Maybe my description is not clear enough.  My intent is to let the code extra owner information from a table property.  In DataHub, we have `User` and `Group` owners.  The Iceberg spec does not provide any guidance on how to set table ownership (could be done with a table property, an entry in the catalog?, or simply inferred from the disk file/folder owner).  Since Apache Spark (and HiveMetaStore?) uses a table property (`owner`), I thought my code should support that way.  My making this configurable, the user could even use 2 different table properties to have users and/or groups as owners.  We talked about this earlier if you remember where Spark would just not allow setting the `owner` property to something else then `${USER}`...  So my org decided to simply use a custom property to set ownership of the table, hence the reason for that code.

Does this help clarifying?